### DISCUSSION TITLE
Football Frenzy

### DISCUSSION THREAD

**Discussion by WhatsJay** [starcoinorg/web3startrek#69] [MoveProjects]
So Guys I really think STC is in Dying need of new project that have both utility and sustainability, in which i believe the token omics and ground work that i have currently endeavored has now achieved. As you can see there is HUGE scalability within the project too moving forward, so not only will we have a community game upon launch in which investors will engage and interact with one another whilst earning an ROI, playing games, entering tournaments and meeting other investors, But we will also upscale this to produce our very own metaverse in which our holders can roam, earn and play.

Take a look at a brief look into the tokenomics and how we wish to launch and upscale this project.
FOOTBALL FRENZY:
Each Mint will cost (X) STC to create.

This will give the buyer 1 Football Frenzy NFT.
Each NFT will have it's own rarity 'league'.
There will be a total of 5 leagues to begin with.
League 1 will contain the rarest NFT's with rarity descending down to league 5.
I plan to not have too much of a separation between the rewards given to the different NFT's based on rarity to ensure all investors can see a ROI and therefore create both a more investable project and sustainable project.
I think first step is source a website developer who can build a website with the following section/channels;

* MYNFT'S - This speaks for it's self, this would be a collection of all owned NFT's.
* MYSTERY BOX'S - Again this is self-explanatory it will consist of all Mystery Box's on sale (Future sales could contain specific box's such as guaranteed league 1-3 box's etc.
* LEAGUES - This would contain the different leagues on show 1-5. for every 500 NFT's Created i suggest we separate them as follows.
League 1 - 50 NFT's
League 2 - 75  NFT's
League 3 - 100 NFT's
League 4 - 125 NFT's
League 5 - 150 NFT's
* Market - A place where people can list there NFT's. I think this should offer Buy/Sell/Swap
Buy - Bid/BuyNow
Sell - Sell
Swap - People can offer there NFT's in Exchange for the NFT on sale and STC either way.
* Swap - This will be an area where STC + Partner coins can be swapped
* Liquidity - This is where STC/Partner Coins can be added to the pool for rewards.
* NFT FLEX - This is where users will be able to view all NFT's in the game/collection.

EXPANION/SCALABILITY
* Develop some game-fi in which users can play 5-a-side football with there nft's against the 'AI' or 'H2H'
* Upscale this into a metaverse where user will use there NFT avatar as there character to roam, play and earn within the metaverse.
* Create added stadiums/kit's etc as NFT's for club merch flex and we could even add ability to merge NFT's kits/stadiums etc
* Create a platform on the website where users can bet on both virtual and real sport events using there STC/Partner tokens.
* Could develop the NFT's to have a 'Top Trump' Effect in which users can stake a set amount of STC to compete against one another and win the staked STC amount. 40% of each mint would goes to INVESTMENT FUND
20% of each mint goes to Daily Rewards Pool
10% of each mint goes to Marketing/Promotion
10% goes to the projects treasury fund
5% of each mint goes to TEAM FUND
5% of each mint goes to INSURANCE FUND 
5% of each mint goes to STC LIQUIDITY POOL to provide stability to both STC and the project its self
5% of each mint goes to Auto Liquidity Exchange

The investment fund will be used to to invest in other projects voted by the community and dev team to then provide more stability to the project through adding 60% of the revenue earned from these projects into the rewards balance. 20% will be re-invested into these projects. 7.5% into STC liquidity pool to further provide stability to both the project and STC. 7.5% will be added to Auto-Liquidity Exchange. Finally 5% will go to Team fund.

I'm currently looking for Developers whom are interested to begin working on this project alongside the team as soon as possible in order to give the well deserved community a project they so rightly deserve.

**Comment by bobnader** [comment]
Well this is a great idea to push the project and bring new investors. Hope you will find a good team to help. Good luck Jack
